src/Servers/Kestrel/Core/src/Internal/Infrastructure/KestrelTrace.cs

```diff
@@ -138,6 +138,14 @@ internal class KestrelTrace : IKestrelTrace
             LoggerMessage.Define<string, Http3ErrorCode>(LogLevel.Debug, new EventId(45, "Http3StreamAbort"),
                 @"Trace id ""{TraceIdentifier}"": HTTP/3 stream error ""{error}"". An abort is being sent to the stream.");
 
+        private static readonly Action<ILogger, string, Http3FrameType, long, long, Exception?> _http3FrameReceived =
+            LoggerMessage.Define<string, Http3FrameType, long, long>(LogLevel.Trace, new EventId(46, "Http3FrameReceived"),
+                @"Connection id ""{ConnectionId}"" received {type} frame for stream ID {id} with length {length}.");
+
+        private static readonly Action<ILogger, string, Http3FrameType, long, long, Exception?> _http3FrameSending =
+            LoggerMessage.Define<string, Http3FrameType, long, long>(LogLevel.Trace, new EventId(47, "Http3FrameReceived"),
```

Comment:
```suggestion
            LoggerMessage.Define<string, Http3FrameType, long, long>(LogLevel.Trace, new EventId(47, "Http3FrameSent"),
```

We should do the same for Http2FrameSent.

Reply:
Should we rename _http3FrameSending to _http3FrameSent then?

Reply:
I'm fine with either "sending" or "sent" as long as we're consistent.

Reply:
IMO sending is more accurate and more consistent 😋